enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/BuiltInProceduresInteractionTestBase.java

```diff
@@ -638,17 +640,38 @@ public void shouldHaveSetTXMetaDataProcedure() throws Throwable
     }
 
     @Test
-    public void shouldFailOnTooLargeTXMetaData() throws Throwable
+    public void readUpdatedMetadataValue() throws Exception
     {
-        ArrayList<String> list = new ArrayList<>();
-        for ( int i = 0; i < 200; i++ )
+        GraphDatabaseFacade localGraph = neo.getLocalGraph();
+        try ( Transaction transaction = localGraph.beginTx() )
```

Comment:
See the example for setTxMetaData in https://github.com/neo4j/neo4j/blob/3.3/enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerIT.java#L157-L164

That looks much cleaner and does not bypass security.